lerna.json

```diff
@@ -3,6 +3,7 @@
   "version": "independent",
   "packages": ["packages/*"],
   "npmClient": "yarn",
+  "tagVersionPrefix": "-v",
   "command": {
     "version": {
```

Comment:
This is not needed anymore right? Cause now release-please is doing changelog and version management?

Reply:
yes, but lerna is still doing the publish. release please uses <component-name>-v<semver> wheras lerna uses <component-name>@<semver> by default. As you can't change the tags as much in release please (only to use the v or not and the component name) i've changed the lerna tags to match the release please tags

Reply:
oh i see, version may not be needed, let me check